runner/src/server/plugins/engine/pageControllers/RepeatingSummaryPageController.ts

```diff
@@ -169,6 +172,11 @@ export class RepeatingSummaryPageController extends PageController {
 
   buildTextFieldRows(answers, form_session_identifier, view = false) {
     const { title = "" } = this.inputComponent;
+
+    form_session_identifier
+      ? `&form_session_identifier=${form_session_identifier}`
+      : "";
```

Comment:
This expression is just being executed on it's own, are we meant to assign this to something?

Reply:
its being assigned further down, which part is being executed on it's own?